streaming/rrdpush.c

```diff
@@ -90,13 +95,20 @@ int rrdpush_init() {
             }
         }
     }
+
     char *invalid_certificate = appconfig_get(&stream_config, CONFIG_SECTION_STREAM, "ssl skip certificate verification", "no");
     if ( !strcmp(invalid_certificate,"yes")){
         if (netdata_validate_server == NETDATA_SSL_VALID_CERTIFICATE){
-            info("The Netdata is configured to accept invalid certificate.");
+            info("The Netdata is configured to accept invalid SSL certificate.");
             netdata_validate_server = NETDATA_SSL_INVALID_CERTIFICATE;
         }
     }
+
+    netdata_ssl_cert_directory = appconfig_get(&stream_config, CONFIG_SECTION_STREAM, "CApath", "/etc/ssl/certs/");
+    netdata_ssl_master_cert = appconfig_get(&stream_config, CONFIG_SECTION_STREAM, "CAfile", "/etc/ssl/certs/");
+    if(!strcmp(netdata_ssl_cert_directory,netdata_ssl_master_cert)) {
+        info("Netdata does not have a SSL master certificate, so it will use the default OpenSSL configuration to validade certificates!");
```

Comment:
validade -> validate.

Reply:
I changed the word and I moved the line for security.c where it makes more sense now.